src/pages/AddedCard.tsx

```diff
@@ -0,0 +1,175 @@
+import { FC } from 'react';
+import { Box, Button, Container, IconButton, Typography } from '@mui/material';
+import { AppFooter } from '~/shared/ui';
+import { Header } from '~/widgets/header';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import CreateIcon from '@mui/icons-material/Create';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+
+interface AddedCardProps {
+  title: string;
+  cardNumber: string;
+  barcodeNumber: number | string;
+}
+
+export const AddedCard: FC<AddedCardProps> = ({
+  title,
+  cardNumber,
+  barcodeNumber,
+}) => {
+  return (
+    <Container
+      sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}
+    >
+      <Header
+        user={{ name: 'иван' }}
+        isLoggedIn={true}
+        type="standard"
+      ></Header>
+      <Box
+        component="div"
+        sx={{
+          minWidth: '20.5rem',
+          margin: 0,
+          padding: '1.5rem 0 1rem',
+          display: 'flex',
+          justifyContent: 'space-between',
+          position: 'relative',
+        }}
+      >
+        {/* Нужно менять на линк из React-Router */}
+        <a
+          href="#"
+          style={{
+            position: 'absolute',
+            fontSize: '0.75rem',
+            color: '#737981',
+            textDecoration: 'none',
+            top: '.3rem',
+            left: '.75rem',
+          }}
+        >
+          Назад
+        </a>
+        <Typography sx={{ fontSize: '2rem' }}>{title}</Typography>
+        <IconButton sx={{ padding: 0 }}>
+          <CreateIcon></CreateIcon>
+        </IconButton>
+      </Box>
+      <Box>
+        <Box
+          sx={{
+            border: '1px solid rgb(73, 69, 78)',
+            width: '20.5rem',
```

Comment:
Так вместе со stretch на уровне выше будет тянуться

```suggestion
            width: '100%',
```